src/main/flatbuffers/rlbot.fbs

```diff
@@ -0,0 +1,188 @@
+
+namespace rlbot.flat;
+
+table ControllerState {
+  /// -1 for full reverse, 1 for full forward
+  throttle:float;
+
+  /// -1 for full left, 1 for full right
+  steer:float;
+
+  /// -1 for nose down, 1 for nose up
+  pitch:float;
+
+  /// -1 for full left, 1 for full right
+  yaw:float;
+
+  /// -1 for roll left, 1 for roll right
+  roll:float;
+
+  /// true if you want to hold the jump button
```

Comment:
I think press makes more sense because this not a toggle field.

But I am okay either way